<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputFileOptions_Dialog</class>
 <widget class="QDialog" name="OutputFileOptions_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>Output File Options</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ferocious.qrc">
    <normaloff>:/images/sine_sweep-32x32-buttonized.png</normaloff>:/images/sine_sweep-32x32-buttonized.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="OutputFileOptions_buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="outFilenameSuffix_lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>(converted)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="FilenameSuffix_checkBox">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>50</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Append this suffix To Output File name:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="SameFileExt_radioButton">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>170</y>
     <width>221</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Default to same file type as Input File</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="setFileExt_radioButton">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>200</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Default to this File Type:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="outFileExt_lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The following options affect the generation of the &lt;span style=&quot; font-style:italic;&quot;&gt;default&lt;/span&gt; output file name:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="outDirectory_lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>select output directory ...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="useOutputDirectory_checkBox">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>110</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Use this output directory:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>78</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Browse ...</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="ferocious.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>OutputFileOptions_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OutputFileOptions_Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OutputFileOptions_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OutputFileOptions_Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
